src/components/LocationInput.test.tsx

```diff
@@ -0,0 +1,22 @@
+import LocationInput from './LocationInput';
+import { screen } from '@testing-library/dom'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+test('renders the component successfully', () => {
+  const { getByText } = render(<LocationInput />)
+  expect(getByText('Enter a Zip Code:')).toBeInTheDocument()
+  expect(getByText('Submit')).toBeInTheDocument()
+
+});
+
+test('types in ui input and handles submit', () => {
+  const locationAddHandler = jest.fn()
+  render(<LocationInput onAddLocation={locationAddHandler}/>)
```

Comment:
Rebase with master. There are new changes that will affect these tests. We are no longer using the add location handler. Right now we are keeping track of the temperature instead which makes more sense for this project